qiskit/qpy/type_keys.py

```diff
@@ -64,6 +64,7 @@
     AlignEquispaced,
 )
 from qiskit.qpy import exceptions
+from qiskit.pulse.configuration import Kernel, Discriminator
```

Comment:
Minor but I would put this alphabetically between the `qiskit.pulse.channels` and `qiskit.pulse.instructions` imports.

Reply:
Done! :) 👉 https://github.com/Qiskit/qiskit-terra/pull/10327/commits/ed7515dd5aff98120824998f3476d1a271c7dbbd